### ISSUE TITLE
No source code ? Closed-source with stolen resources

### ISSUE THREAD

**Issue by yuroyami** [Chaintech-Network/CMPCharts#4]
Another library with closed-source code possibly "taken" from https://github.com/patrykandpatrick/vico with no mention whatsoever to the original authors.
Violating open-source licenses tenfold.

**Comment by yuroyami** [comment]
Nope, just a hunch. Since your code is "not visible" its probably not even your work.

**Comment by ctn-mobile-innovation** [comment]
Wow!
